### ISSUE TITLE
Question about pretrain loss?

### ISSUE THREAD

**Issue by hzlllll** [bowang-lab/scGPT#30]
Thanks for your work! I have noticed that the second version of the preprint has been published, but I still have the question about the pretrain loss in Section 4.3.3. How much parts are the loss of pretrain composed?  Is it equals to the sum of gene expression prediction loss and  cell prompt loss or how to combine the gene prompt loss and cell prompt loss during the pretraining?

**Comment by subercui** [comment]
Hi, thank you for your interest! We did the gene prompt step and cell prompt step consecutively during the pre-training. The gene prompt step is to predict the expression values of unknown genes based solely on known genes, and the cell prompt step is to predict unknown genes based on the cell embedding as well.

We are wrapping up the code for pretraining and data collection right now, and aim to release them by the end of this week. Hopefully, that will explain this question as well.

**Comment by hzlllll** [comment]
Thanks for your answers!
